### PR TITLE
Kubernetes 1.18 feature Windows CSIproxy (alpha) blog

### PR DESCRIPTION
**Pull request by pgburt**
/hold
/assign @karenhchu

### CODE REVIEW

**Comment by pgburt** [comment]
/assign @onlydole
/assign @mrbobbytables 

As an aside, it looks like something funky is going on with the second and third image on the blog. They're significantly taller and wider than they ought to be. Any thoughts on how to best crop/edit them down?

**Comment by mrbobbytables** [comment]
The images themselves look small when being previewed:
https://deploy-preview-20069--kubernetes-io-master-staging.netlify.com/blog/

I'd need to pull the branch down and play around to investigate

**Comment by pgburt** [comment]
Seeing if I can get the CLA to pass by noting:
I signed it

**Comment by onlydole** [comment]
Howdy @pgburt - on those images, it looks like they might be able to be cropped a bit more. I see a good deal of whitespace around the borders might be one issue. I can take a look if you see anything beyond that in an hour or so.

**Comment by karenhchu** [comment]
Can we get a LGTM @onlydole  @mrbobbytables @castrojo? :) thank you!

File: content/en/blog/_posts/2020-04-03-Introducing-Windows-CSI-support-alpha-for-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+date: 2020-04-03
+slug: kubernetes-1-18-feature-windows-csi-support-alpha
+---
+**Authors:** Authors: Deep Debroy [Docker], Jing Xu [Google], Krishnakumar R (KK) [Microsoft]
+
+<em>The alpha version of [CSI Proxy][csi-proxy] for Windows is being released with Kubernetes 1.18. CSI proxy enables CSI Drivers on Windows by allowing containers in Windows to perform privileged storage operations.</em>
+
+## Background
+
+Container Storage Interface (CSI) for Kubernetes went GA in the Kubernetes 1.13 release. CSI has become the standard for exposing block and file storage to containerized workloads on Container Orchestration systems (COs) like Kubernetes. It enables third-party storage providers to write and deploy plugins without the need to alter the core Kubernetes codebase. All new storage features will utilize CSI, therefore it is important to understand what a CSI plugin does.
```
[the remark this answers is not in the document]
Comment (pgburt) [reply content/en/blog/_posts/2020-04-03-Introducing-Windows-CSI-support-alpha-for-Kubernetes.md]: Changed, and pushed the new version. Give it a look.

**Comment by onlydole** [comment]
You bet, @karenhchu!
/lgtm

**Comment by karenhchu** [comment]
Can someone approve it? @mrbobbytables @castrojo? Almost there!

**Pull request merged**